Parser/pegen/pegen.h

```diff
@@ -119,6 +130,7 @@ void *_PyPegen_indent_token(Parser *p);
 void *_PyPegen_dedent_token(Parser *p);
 expr_ty _PyPegen_number_token(Parser *p);
 void *_PyPegen_string_token(Parser *p);
+void *_PyPegen_type_comment_token(Parser *p);
```

Comment:
What if we return a `Token *` here, so that we can avoid the cast in the `#define`?

Reply:
Heh, this function no longer exists. Nor do most of the others.

Reply:
This was showing as added in the Github UI. Sorry.

Reply:
Well, I did add it, but then I merged master and you had done something to make it unnecessary. :-)

Reply:
Understood now, that was @pablogsal in #19775.